libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -161,7 +161,7 @@ class JSONNode {
 
 
     json_string as_string(void) const json_nothrow json_read_priority;
-    int as_int(void) const json_nothrow json_read_priority;
+    json_int_t as_int(void) const json_nothrow json_read_priority;
```

Comment:
libjson supports just one integer type, which is configurable. Currently we have `long long`. On all our architectures it is 64 bit. I don't know of a single non-embedded platform where `long long` is not 64 bit. But I don't care too much.